README.md

```diff
@@ -750,7 +751,18 @@ Available methods:
     status = apiaudio.Connector.connection(
       connection_id="af2fe14a-aa6b-4a97-b430-a072c38b11ff"
     )
-    
+
+### Logging <a name = "logging"></a>
+
+By default, warnings issued by API are logged in the console output. Additionally, some behaviors are logged on informational level (e.g. "In progress..." indicators during longer processing times). 
+The level of logging can be controlled choosing from the standard levels in Python's `logging` library.
```

Comment:
The level of logging can be controlled by choosing from the standard levels in Python's `logging` library.